apicli/version.py

```diff
@@ -0,0 +1,5 @@
+from distutils.version import StrictVersion
+
+__all__ = ['VERSION']
+
+VERSION = StrictVersion('0.0.1')
```

Comment:
Looks like the convention is `__version__`: http://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package